### DISCUSSION TITLE
Starting in offline-mode

### DISCUSSION THREAD

**Discussion by bstro** [rocicorp/replicache#1044] [Q&A] [answered]
Reading the docs at https://doc.replicache.dev/concepts/offline 
It sounds as though it is not possible for replicache to start in an offline mode:
> For the same reason as above, if a tab is closed while offline then a new tab is started while still offline, the new tab will not see changes the first tab made. We do not recommend / support using Replicache with apps that can be started fully offline (e.g., via service worker).

This feature is exactly what I need for my project. I would feel comfortable starting with replicache and getting the functionality when it ships. I am looking for some signal for the priority of this feature for this project.

Thanks!

**Comment by aboodman** [comment, marked as answer]
Sorry for the slow response @bstro. Yes, we plan to ship the feature in the next few months. It's already implemented, just waiting on cycles to document the upgrade process.

It's not the most important thing we're working on, but it is something we're going to do. HTH.
